nikola/nikola.py

```diff
@@ -1117,6 +1122,7 @@ def _set_global_context_from_config(self):
         self._GLOBAL_CONTEXT['generate_atom'] = self.config.get('GENERATE_ATOM')
         self._GLOBAL_CONTEXT['generate_rss'] = self.config.get('GENERATE_RSS')
         self._GLOBAL_CONTEXT['rss_path'] = self.config.get('RSS_PATH')
+        self._GLOBAL_CONTEXT['rss_filename_base'] = self.config.get('RSS_FILENAME_BASE')
```

Comment:
*_extension too. Or otherwise make them cause full rebuilds. (We should have a facility for that, that isn’t global context.)

Reply:
Fixed. I also added documentation for them.